tests/raftstore/cluster.rs

```diff
@@ -313,23 +315,34 @@ impl<T: Simulator> Cluster<T> {
                    reqs: Vec<Request>,
                    timeout: Duration)
                    -> RaftCmdResponse {
-        let mut try_cnt = 1;
+        let mut try_cnt = 5;
         loop {
             let mut region = self.get_region(key);
             let region_id = region.get_id();
             let req = new_request(region_id, region.take_region_epoch().clone(), reqs.clone());
-            let resp = self.call_command_on_leader(region_id, req, timeout).unwrap();
-            if resp.get_header().has_error() {
-                if self.refresh_leader_if_needed(&resp, region_id) {
-                    warn!("seems leader changed, let's retry");
-                    continue;
-                } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
-                    warn!("seems split, let's retry");
-                    try_cnt += 1;
-                    continue;
+            let result = self.call_command_on_leader(region_id, req, timeout);
+            match result {
+                Ok(resp) => {
+                    if resp.get_header().has_error() {
+                        if self.refresh_leader_if_needed(&resp, region_id) {
+                            warn!("seems leader changed, let's retry");
+                            continue;
+                        } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
+                            warn!("seems split, let's retry");
+                            try_cnt += 1;
+                            continue;
+                        }
+                    }
+                    return resp;
+                }
+                Err(e) => {
+                    if let Error::Timeout(_) = e {
```

Comment:
that's because we map it to Other(String) error previously...I've remove that line

        .map_err(|e| box_err!("call command failed {:?}, cost {:?}", e, t.elapsed()))